index.html

```diff
@@ -60,6 +63,80 @@
           }],
 
           localBiblio: {
+            "IMSC1.0": {
+              authors: [
+                  "Pierre-Anthony Lemieux"
+              ],
+              title: "TTML Profiles for Internet Media Subtitles and Captions 1.0.1 (IMSC1)",
+              href: "https://www.w3.org/TR/2018/REC-ttml-imsc1.0.1-20180424/",
+              publisher: "W3C",
+              status: "REC",
+              date: "24 April 2018"
+            },
+            "IMSC1.1": {
```

Comment:
I am completely opposed to making any use of specref references for the simple reasons that (1) they can (and do) change from under us without notice, thus making any document that employs them unstable, and (2) they do not contain the same information that we employ in our primary TTML specifications. We can discuss this further in teleconference.

Reply:
Using Specref is normal practice in W3 specs. If you want to change the entries, or have visibility of proposed change, this is done at https://github.com/tobie/specref .

Reply:
That does not fix the document stability issue. This is really a pure editorial issue in the domain of the editors. Our choice is to use inline definitions to avoid instability due to unexpected and unannounced changes in specref over which  we have no editorial control.

Reply:
That's not correct: the document is stable with respect to W3C specifications because the W3C /TR space spec info provided by specref is sourced from https://www.w3.org/2002/01/tr-automation/tr.rdf . If you think the Specref information isn't correct then we need to raise an issue with the team who maintain tr.rdf (or the tool that updates it on spec publication).

Reply:
I disagree. The sprecref data is not under direct control of the editors of this document nor is under the process control of this group; therefore, it is unstable and unfixed. If you like, we can simply convert this document to HTML5 or XML Doc format and stop using respec, but I will not agree to use respec if it forces us to use specref data to resolve bibliographic data. I note once again that this decision is in the domain of the editors (absent formal TTWG policy to the contrary).

Reply:
In my view consistency across W3C documents takes precedence over consistency across TTWG documents; since the data is provided by W3C, we should use it.  Furthermore, I've just made the change locally to switch all the W3C spec references into aliases and the only differences I can see are:

1. the dates do not consistently zero-pad the day part to a two digit number in the text description of the document being referenced.
2. for SDP-US the phrase "W3C Note" becomes "W3C Working Group note"

So there does not seem to be a substantive problem with changing this, and we could consider raising a query about the date format.

Since I've made the change, I'd be happy to push it directly if you'd withdraw your objection @skynavga ?

Reply:
No, I do not withdraw my objection. As you have shown, even your attempt to synchronize the content was unsuccessful. The larger issue for me is that you are personally attempting to impose editorial control on a document for which you are not an editor, and you are blocking progress on the part of the editor's work.

Reply:
> As you have shown, even your attempt to synchronize the content was unsuccessful.

I drew the opposite conclusion from the minor nature of the deltas! Essentially, I thought my attempt was successful and that the differences were so small as to be not worth holding out for.

Regarding "editorial control" I disagree, but as you've noted before, Github spec repository pull requests and issues aren't a good place to discuss process.

Reply:
I have looked at specref.org. It is not affiliated with the W3C and appears to impose copyright status on the content that may be used for specification data. As such, and here I am speaking with both my editor and AC representative hats, I cannot accept content that is arbitrarily inserted into our working group documents for which we as a TTWG and the W3C itself has no control and may be in violation of IPR policies.

Reply:
@plehegar can you please provide guidance about https://github.com/w3c/tt-profile-registry/pull/55#discussion_r251211105 ? Specref seems to be used endemically in W3C specs and that seems like a good thing to me, but @skynavga is raising an issue that it is not suitable for use in this way.

A couple of other points worth considering @skynavga :
1. When the spec gets published to /TR the references etc are frozen, i.e. it's the respec formatted output that is uploaded, not the version with the respec script running. This means that even if the reference changes in Specref it won't magically break/fix documents in /TR.
2. Specref cannot claim copyright on the data it is providing, which is clearly owned by whatever organisations are publishing the specifications themselves. The only way it might have copyright is if the information is deliberately incorrect, which I believe is never the case.